_posts/2021-05-16-overview-of-architecture-review-processes.md

```diff
@@ -0,0 +1,253 @@
+The ADR form a "decision log" {% cite ADR2021 %}.
+
+For more information, we refer to [https://adr.github.io/](https://adr.github.io/).
+
+### Parties
+
+When working with ADR, we can distiguish the following parties:
+  1. The project team/creator(s) of an ADR,
+  1. The ADR reviewer(s)
+
+### Process
+
+When there is an architecturally significant decision to be made, the project team, the involved architect, or - in general - persons who are involved in the decision making create an ADR.
+Based on Nygard's proposal, ADR typically contain 5 sections {% cite Nygard2011 %}:
+  * Title: It is typically describing the architecture decision shortly.
+  * Status: May be "Proposed", "Request for Comment", "Accepted", or "Superseded".
+  * Context: The reasons of the decision to be made.
+  * Decision: A description of the decision itself and its justification.
+  * Consequences: The impact of the decision such as the benefits and downsides regarding architecture characteristics, etc.
+
+In general, it is a good idea to let others review an ADR.
+For that, you use the status of the ADR.
+A newly created ADR, then, can be in status "Proposed".
+When you want other to review the ADR, you can set the status to "Request for Comment".
+The ADR can be reviewed by other parties such as other project members, other teams, the (lead) architect, the architecture guild, or persons in the company who are good in architecture work.
+As soon as the review is done, the ADR status can be changed to "Accepted" and should be stored somewhere at a public location.
+
+ADR can be stored in GIT or in a wiki.
+There are tools for managing ADR in GIT, e.g., [https://github.com/npryce/adr-tools](https://github.com/npryce/adr-tools).
+In my personal opinion, it is recommendable to store ADR in a public location such as a wiki for better accessibility from an potentially interested audience.
+However, the tool should be able to track changes to the ADR in a history to identify authors and revert changes if needed.
+Moreover, a comment functionality for reviewing an ADR is beneficial.
+
+Every ADR and, thus, every decision is an own small document (see also: "Decision log" in the previous section).
+ADR can get an ID and be referenced in other documentation or wiki pages.
+New ADR can change aspects of previous ADR.
+If that happens, they can change the status of the previous ADR to "Superseded".
+
+### Artifacts
+
+The ADR itself is the only artifact in this prototypical architecture review process.
+Reviews on the ADR should be done on the ADR itself when the ADR is created.
+
+In recent years, different templates have appeared for ADR.
+In {% cite Henderson2021 %}, Henderson gathered different ADR example templates.
+However, ADR can be combined with other ways to document architectures such as Design Documents (see also: Design Document or Lightweight Request for Comment Approach):
```

Comment:
> see also: Design Document or Lightweight Request for Comment Approach)

We are on the web. Maybe adding an anchor link? ;)

Reply:
I tried to set up anchors. Was not successful so far. But I will reconsider that :-)